frontend/src/scenes/billing/BillingProduct.tsx

```diff
@@ -331,7 +347,20 @@ export const BillingProduct = ({ product }: { product: BillingProductV2Type }):
                                                     <LemonButton
                                                         status="stealth"
                                                         fullWidth
-                                                        onClick={() => deactivateProduct(product.type)}
+                                                        onClick={() => {
+                                                            posthog.getActiveMatchingSurveys((surveys: Survey[]) => {
+                                                                const matchingSurvey = surveys.filter(
+                                                                    (survey) => survey.name === 'Open feedback'
```

Comment:
Why not grab the churn survey's ID from the app and use that instead? It should be exposed when you create an API survey :o